call-for-champions.md

```diff
@@ -0,0 +1,97 @@
+# Call for Turing Reproducible Research Champions
+**Application deadline:** Friday 23rd February 2018
+
+## What is a Reproducible Research Champion?
+We're looking for Turing researchers who want to take their research to the
+next level in terms of reproducibility and help us establish a culture of
+reproducible research at the Turing.
+
+Enthusiasm for reproducible research is the most important factor. There are many
+reasons that researchers who believe in reproducible research struggle to make
+their research reproducible in practice, and Champions will receive significant
+support from the Turing Research Engineering team to overcome these barriers to
```

Comment:
Research Software Engineering team? Did you drop the software on purpose? For length?

I don't really mind TBH, just curious :smile:

Reply:
We're now 50% RSEs and 50% Data Scientists so needed something to cover both. Happy to change to "support from Turing Research Software Engineers" if that appeals more / avoids a speedbump when parsing.

Reply:
:tada: - no need to change - thanks for explaining :)